### PR TITLE
lint samples

### PR DESCRIPTION
**Pull request by benmccann**

### CODE REVIEW

**Comment by benmccann** [comment]
@jledentu would you be able to approve this PR for me? I can't merge any code until it has at least 1 approval. Thanks!

File: .htmllintrc [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+	"indent-style": "tabs",
+	"line-end-style": false,
+	"attr-quote-style": "double",
```
Comment (jledentu) [inline .htmllintrc]: Is this needed, since `double` is the default value of `attr-quote-style`?

Comment (benmccann) [reply .htmllintrc]: removed. good catch. I just copied the `.htmllintrc` file from one of the other chart.js repos. I guess we could remove it from the others as well

**Comment by benmccann** [comment]
Thanks @jledentu!

**Pull request merged**